### DISCUSSION TITLE
Setting the number of lines to trigger scrolling

### DISCUSSION THREAD

**Discussion by Shamaoke** [davatorium/rofi#1540] [Q&A]
Hi.

When using the continuous scrolling method (`scroll-method: 1`), the window starts scrolling at the 10th entry (in the case of the number of lines of the Rofi window is 20). Is there any way to configure this behaviour and set the number of lines to trigger scrolling (like, for example, in the Vim editor when using the `scrolloff` setting)?

Thanks.

**Comment by DaveDavenport** [comment]
no option like this exists.
